### ISSUE TITLE
macOS Accessibility incompatibility

### ISSUE THREAD

**Issue by James6M** [therealglazou/bluegriffon#70]
On macOS 10.14.6, I'm getting extremely frequent BlueGriffon crashes, to the point that the software is unusable.  The Problem Report always says "due to uncaught exception 'NSAccessibilityException' ".  I'm interested in work-arounds as well as fixes.  As far as I can tell, the Accessibility components are too baked-in to the OS to turn them off.

**Comment by James6M** [comment]
I've narrowed the failure modes, at least the ones that I'm experiencing.  Crashes are in conjunction with two third-party utilities which use the Accessibility components to control windows.  Having [Smart Scroll](https://www.marcmoini.com/sx_en.html) active results in a BlueGriffon crash whenever the mouse pointer enters a BlueGriffon window.  Having [Zooom/2](https://zooom.en.softonic.com/mac) active results in a BlueGriffon crash on any action which tries to open a sheet from the title bar (e.g., File> Open or Format> Page Properties).
